iOS/Flows/Conversation/Cells/MessageCell.swift

```diff
@@ -87,11 +86,14 @@ class MessageCell: UICollectionViewCell {
         }
     }
 
+    // MARK: Configuration
+
     func configure(with message: Messageable) {
         self.content.configure(with: message)
         
         self.messageState.message = message
         self.messageState.deliveryStatus = message.deliveryStatus
+        self.messageState.statusText = message.context.displayName
```

Comment:
Make sure the status text is configured when a new message is assigned for this cell. This fixes cell recycling issues.